node/src/effect/requests.rs

```diff
@@ -469,7 +485,16 @@ impl Display for StorageRequest {
     fn fmt(&self, formatter: &mut Formatter<'_>) -> fmt::Result {
         match self {
             StorageRequest::PutBlock { block, .. } => write!(formatter, "put {}", block),
+            StorageRequest::PutBlockAndDeploys {
+                block: block_deploys,
+                ..
+            } => {
+                write!(formatter, "put block and deploys {}", block_deploys)
```

Comment:
Maybe we should log just the hask of a block rather than the whole block?

Reply:
Fixed in [02645db](https://github.com/casper-network/casper-node/pull/3010/commits/02645db01331a35cc2ab337177741ca0ffae3f3f).